common/src/main/java/com/wynntils/core/chat/tabs/ChatTabModel.java

```diff
@@ -74,6 +86,14 @@ public static void setFocusedTab(ChatTab focused) {
             chatTabData.putIfAbsent(focusedTab, new ChatComponent(McUtils.mc()));
             unreadMessages.put(focusedTab, false);
             McUtils.mc().gui.chat = chatTabData.get(focusedTab);
+
+            // If chat screen is open, and current message is empty, set our auto command
```

Comment:
I like your solution. But this comment is slightly misleading now. Fix it and then you can just push it.

Reply:
```suggestion
            // If chat screen is open, and current message is empty or the previous auto command, set our auto command
```

Reply:
I fixed it for ya :)